rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc

```diff
@@ -49,9 +49,9 @@ include::modules/rosa-sts-overview-of-the-default-cluster-specifications.adoc[le
 //include::modules/rosa-sts-associating-your-aws-account.adoc[leveloffset=+2]
 include::modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc[leveloffset=+1]
 [id="rosa-hcp-creating-vpc"]
-= Creating a Virtual Private Cloud for your ROSA {hcp} clusters
+== Creating a Virtual Private Cloud for your {hcp-title} clusters
 
-You must have a Virtual Private Cloud (VPC) to create {hcp} for ROSA cluster. You can manually create these using the AWS console, or you can use `terraform` to create a VPC by using a template.
+You must have a Virtual Private Cloud (VPC) to create {hcp-title} cluster. You can manually create these using the AWS console, or you can use `terraform` to create a VPC by using a template.
 
 include::modules/rosa-hcp-vpc-terraform.adoc[leveloffset=+2]
 
```

Comment:
I think this applies to the module at line 58. Table 3 (Requirements for your VPC) capitalizes "availability zones" in the second column. None of our other documentation capitalizes those words. I'm also not sure about the spelling out numbers rule like the other table.

Reply:
On this:
> 70: I am also not sure if "Red Hat-hosted cluster" down at line 70 needs to be changed to something like "Red Hat hosted control planes cluster" or not....?

Red Hat hosts the cluster itself. Hosted control planes are, confusingly, different.

> I still don't like "calls" because it sounds a little too close to personification, but I don't know what a better words is. Would "is programmed to" or something like that work. Honestly have no idea.

Unfortunately, a call is a baked in term, so I think it still fits, but I do take that it isn't a great phrasing.